spec/controllers/alchemy/admin/pages_controller_spec.rb

```diff
@@ -598,6 +598,32 @@ module Alchemy
             alchemy_get :edit, id: page.id
           end
         end
+
+        context 'if page is scoped' do
+          context 'to a single role' do
+            context 'user role does not match' do
+              before do
+                expect_any_instance_of(Page).to receive(:editable_by?).at_least(:once) { false }
+              end
+
+              it 'redirects to sitemap' do
+                alchemy_get :edit, id: page.id
+                expect(response).to redirect_to(admin_dashboard_path)
```

Comment:
@Brantron is right, this is default Alchemy authorization behavior. I t don't think, though, that we need to spec this here then. Happy path from below is enough.